doc/api/cli.md

```diff
@@ -121,6 +121,14 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--redirect-warnings`
+<!-- YAML
+added: REPLACEME
+-->
+
+Write process warnings to the given file instead of printing to stderr but
+only if the file does not already exist.
```

Comment:
I'm not entirely sure about the `only if the file does not already exist` part either. For now I have it this way just to ensure that something important isn't accidentally overwritten.